### DISCUSSION TITLE
"telnet: Unable to connect to remote host: Cannot assign requested address" for Port 1883 but not for 22...?

### DISCUSSION THREAD

**Discussion by jayjupdhig** [emqx/emqx#5255] [Q&A]
As described here:

https://translate.google.com/translate?hl=en&sl=auto&tl=en&u=https%3A%2F%2Fdebianforum.de%2Fforum%2Fviewtopic.php%3Ff%3D27%26t%3D181488%26p%3D1276749%23p1276749

Thank you very very much for your feedbacks! :-)

**Comment by k32** [comment]
Hello,

It looks like you don't have a service listening on port 1883, but have sshd listening on port 22, and also have ipv6 disabled, so when telnet fails to connect to the IPv4 address and tries to fall back to IPv6 address, it fails to open connection.

Could you provide the following information:
1) `cat /proc/net/if_inet6`
2) `lsmod` 
3) `strace telnet localhost 1883`

**Comment by k32** [comment]
You are likely seeing this issue because EMQ X deamon is not running, and there is no listener on port 1883, while your OS is not configured to use IPv6.
